services/ml/compilation_delegate_dnnl.cc

```diff
@@ -544,10 +549,9 @@ int32_t CompilationDelegateDnnl::AddReorder(const std::string& input_name,
   return mojom::NOT_ERROR;
 }
 
-int32_t CompilationDelegateDnnl::AddFusedActivation(
-    const std::string& input_name,
-    const std::string& output_name,
-    int32_t fuse_code) {
+int32_t CompilationDelegateDnnl::AddActivation(const std::string& input_name,
+                                               const std::string& output_name,
+                                               ActivationType activation_type) {
```

Comment:
I apologize that I was not clear. But I think `fuse_code` and `type` could be one parameter, they are only `relu`, `relu1`, `relu6` and `logitics`, are they?

Reply:
I understand your suggestions now. New code has been pushed. Please help to review again. Thanks!